code/modules/mob/living/silicon/damage_procs.dm

```diff
@@ -1,5 +1,5 @@
 
-/mob/living/silicon/apply_damage(damage = 0,damagetype = BRUTE, def_zone = null, blocked = FALSE, forced = FALSE)
+/mob/living/silicon/apply_damage(damage = 0,damagetype = BRUTE, def_zone = null, blocked = FALSE, forced = FALSE, spread_damage = FALSE)
```

Comment:
why did you add this

Reply:
linter complained about the spread_damage argument not being there, so i added it. Might also prevent issues down the line, anyway.